community/lucene-index/src/test/java/org/neo4j/kernel/api/impl/schema/DatabaseCompositeIndexAccessorTest.java

```diff
@@ -152,24 +152,6 @@ public void indexReaderShouldSupportScan() throws Exception
     }
 
     @Test
-    public void indexReaderShouldSupportSeekOnCompositeIndexUpdatedWithTwoTransactions() throws Exception
-    {
-        // GIVEN
-        updateAndCommit( asList( add( nodeId, values ), add( nodeId2, values ) ) );
-        updateAndCommit( asList( add( nodeId, values2 ), add( nodeId2, values2 ) ) );
-        IndexReader reader = accessor.newReader();
-
-        // WHEN
-        PrimitiveLongIterator results = reader.query( IndexQuery.exists( PROP_ID1 ), IndexQuery.exists( PROP_ID2 ) );
-
-        // THEN
-        assertEquals( asSet( nodeId, nodeId2 ), PrimitiveLongCollections.toSet( results ) );
-        assertEquals( asSet( nodeId ), PrimitiveLongCollections.toSet( reader
-                .query( IndexQuery.exact( PROP_ID1, values[0] ), IndexQuery.exact( PROP_ID2, values[1] ) ) ) );
-        reader.close();
-    }
```

Comment:
No, not as such. This test does not work. A test written in this way can never test what this test is trying to test, as this works directly with `IndexEntryUpdates`. A test for this would need to use `NodeUpdates`.